src/Func.h

```diff
@@ -2281,6 +2281,12 @@ class Func {
      */
     Func &async();
 
+    /** Adds an extra dimension to the function storage to enable
```

Comment:
So is the idea that if I specify extent=N, I end up with N times the amount of storage allocated, partitioned into a ring buffer? If so, let's document that.